zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/encoding/AuthInfo.java

```diff
@@ -73,25 +90,25 @@ public DirectBuffer toDirectBuffer() {
   }
 
   public Map<String, Object> toDecodedMap() {
-    switch (getFormat()) {
-      case JWT -> {
-        final String jwtToken = getAuthData();
-        return Authorization.jwtDecoder(jwtToken)
-            .withClaim(Authorization.AUTHORIZED_TENANTS)
-            .decode();
-      }
-      default -> {
-        return Map.of(
-            Authorization.AUTHORIZED_TENANTS, List.of(TenantOwned.DEFAULT_TENANT_IDENTIFIER));
-      }
+    if (Objects.requireNonNull(getFormat()) == AuthDataFormat.JWT) {
```

Comment:
:upside_down_face: I don't think the `requireNonNull` is doing much here, a normal NPE will show you the same (or actually better?) information.